src/lib/create-combobox.ts

```diff
@@ -309,10 +352,12 @@ export function createCombobox<T extends Item>({
     const controller = new AbortController();
     node.addEventListener("blur", close, { signal: controller.signal });
     node.addEventListener("focus", open, { signal: controller.signal });
-    node.addEventListener("click", open, { signal: controller.signal });
```

Comment:
I removed this because it doesn't match the way other comboboxes handle clicking when already focused. Tested and all of our other triggers to open the list match downshift, etc.